sdk/servicebus/azure-servicebus/tests/test_mgmt_queues.py

```diff
@@ -0,0 +1,174 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+import pytest
+
+from azure.core.exceptions import HttpResponseError, ServiceRequestError
+from azure.servicebus.management import ServiceBusManagementClient
+from azure.servicebus import ServiceBusSharedKeyCredential
+
+from devtools_testutils import AzureMgmtTestCase, CachedResourceGroupPreparer
+from servicebus_preparer import (
+    CachedServiceBusNamespacePreparer
+)
+
+
+class ServiceBusManagementClientQueueTests(AzureMgmtTestCase):
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    def test_list_queues_by_conn_string(self, servicebus_namespace_connection_string):
```

Comment:
One nitpick:  Let's centralize on a prefix so we can run tests easily via -k.  e.g. "test_mgmt_" with the substrata "test_mgmt_queue" such that this would be "test_mgmt_queue_list_by_conn_string"  (I don't even know if we need the "by conn string" part, that seems a holdover from old tests, and I would only note the auth type if the purpose of the test were to evaluate the auth type; so I might go as far as "test_mgmt_queue_list")

Reply:
done done